docs/cow-protocol/tutorials/solvers/cow-amm.md

```diff
@@ -63,35 +38,35 @@ You need to choose a valid CoW Swap order with the following restrictions:
 - `sellTokenBalance`: must be `BALANCE_ERC20`.
 - `buyTokenBalance`: must be `BALANCE_ERC20`.
 
+:::note
+
+The sell and buy amount specified are the limit amounts of the order. The actual executed amounts are determined using prices from the auction.
+In that regard, CoW AMM orders behave exactly like normal trader orders.
+
+:::
+
 You also need to compute:
 - the order hash `hash` as defined in the library `GPv2Order`, and
-- the order signature (example code that generates a valid signature is [the `getTradeableOrderWithSignature` function in the ComposableCoW repo](https://github.com/cowprotocol/composable-cow/blob/main/src/ComposableCoW.sol)).
+- the order signature
 
-This order can be included in a batch as any other CoW Protocol orders with three extra conditions:
-- One of the pre-interaction must set the commitment by calling `ConstantProduct.commit(hash)`.
-- Must contain at most one order from the AMM in the same batch.
-- One of the post-interactions must reset the commitment by calling `ConstantProduct.commit(EMPTY_COMMITMENT)`.
+Example code that generates an order can be found in `order` function of the BCoWHelper contract
```

Comment:
Can we link here the factory and helper contract deployment addresses here (cf. https://cowservices.slack.com/archives/C035TDPHQAJ/p1723126963478129?thread_ts=1723034842.053739&cid=C035TDPHQAJ)

Reply:
I added a link to the code of contract.